wrappers/golang/curves/bls12377/ecntt/ecntt_test.go

```diff
@@ -0,0 +1,33 @@
+package ecntt
+
+import (
+	"testing"
+
+	"github.com/ingonyama-zk/icicle/wrappers/golang/core"
+	. "github.com/ingonyama-zk/icicle/wrappers/golang/curves/bls12377"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	largestTestSize = 17
+)
+
+func TestECNtt(t *testing.T) {
```

Comment:
Since this is a separate package from bls12377, this test needs its own `initDomain` call since the bls12377 package will releaseDomain in the teardown portion of its tests.